cmd/update-rules/main.go

```diff
@@ -86,8 +91,7 @@ func main() {
 	}
 
 	// update rules for all destination repos
-	UpdateRules(rules, o.branch, o.goVersion)
-
+	UpdateRules(rules, o.branch, o.goVersion, o.remove)
```

Comment:
Is it better to have one more args here, or pass the options all together?  WDYT. We are not expecting much changes. So I think this should be good enough